juriscraper/pacer/reports.py

```diff
@@ -198,6 +199,11 @@ def download_pdf(
                 "caseid": pacer_case_id,
                 "magic_num": pacer_magic_num,
             }
+            if appellate:
+                url = make_docs1_url(self.court_id, pacer_doc_id)
+                params = {
+                    "uid": pacer_magic_num,
+                }
```

Comment:
Should there be an `else` here for the normal path, so that the `if appellate` branch doesn't get the `magic_num` and `caseid` params?

Reply:
In this case, I was overriding the params completely before passing to the request so in case it's appellate only passes the `uid` without the `magic_num` or the` caseid`. But yes I think it's more clear using the `else`, so I added it.